style.css

```diff
@@ -23,13 +27,68 @@ ul {
 	list-style: none;
 }
 
-a {
+a:not(.item-link) {
 	display: block;
 	text-align: center;
-	color: #a6a2c5;
-	text-transform: uppercase;
+	color: var(--color);
+	padding: 15px;
+	text-decoration: none;
+	font-size: 25px;
+}
+
+.item-link {
 	padding: 15px;
 	text-decoration: none;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.item-icon {
+	margin-bottom: 5px;
+	transform: translateY(0px);
+	transition-duration: 450ms;
+}
+
+.item-text {
 	font-size: 25px;
+	color: var(--color);
+	font-family: sans-serif;
+	position: absolute;
+	opacity: 0;
+	transform:translateY(0) ;
+	transition-duration: 500ms;
 }
 
+.item-dot {
+	display: block;
+	width: 8px;
+	height: 8px;
+	border-radius: 50%;
+	background-color: var(--color);
+	opacity: 0;
+	transform:scale(0);
+	transition:all 500ms;
```

Comment:
Avoid using `all`. This animates everything, even things that you don't need and can make a transition slower. Always be specific about the things you really want to animate.